test/token.js

```diff
@@ -61,7 +61,7 @@ contract("Token", accounts => {
     });
 
     it("should NOT be able to transfer more tokens than they have", async () => {
-      await checkErrorRevert(etherRouterToken.transfer(ACCOUNT_TWO, 1500001));
+      await checkErrorAssert(etherRouterToken.transfer(ACCOUNT_TWO, 1500001));
```

Comment:
I don't think I exactly understand why these have gone from `revert`s to `assert`s?

Reply:
Because the old `EtherRouter` did a `require` check on the success of the `delegatecall` success. If that failed, even for an `assert` we still got a `require` failure

Reply:
So the require check has been replaced by a `switch`. If it fails, it then does a `revert`, which is what a `require` does, right, and so the behaviour should be the same?

Reply:
I take that bit back, you're right. I am still not 100% sure what is going on here but the history is that when the ganache version switched, this somehow broke the way failed asserts behaved when executed via `EtherRouter` which necessitated these changes: https://github.com/JoinColony/colonyNetwork/commit/780227a65c84a18a2e7a59ef3472fc86acb9ecd7#diff-6af24f2316d75cbe66c8a65032a385ba

Reply:
And before that when we switched the EtherRouter from using an `assert` to `require` those `checkErrorAssert`s on Token test worked https://github.com/JoinColony/colonyNetwork/pull/129/files#diff-3776618b80cc8f554676f40c34ae0c9f